head/firmware/can_task.cpp

```diff
@@ -81,6 +81,15 @@ struct motion_controller::HardwareConfig PinConfigurations {
         .port = GPIOC, .pin = GPIO_PIN_4, .active_setting = GPIO_PIN_SET},
 };
 
+struct motion_controller::HardwareConfig PinConfigurations2 {
```

Comment:
can we give these better names? `pin_configurations_left` and `pin_configurations_right` maybe?